pkg/segment/query/segqueryhelpers.go

```diff
@@ -65,6 +65,14 @@ type QuerySegmentRequest struct {
 	TotalRecords         uint32
 }
 
+func (qsr *QuerySegmentRequest) GetStartEpochMs() uint64 {
+	return qsr.segKeyTsRange.StartEpochMs
+}
+
+func (qsr *QuerySegmentRequest) GetEndEpochMs() uint64 {
+	return qsr.segKeyTsRange.EndEpochMs
+}
+
```

Comment:
nit: move these down near the toerh QuerySegmentRequest getters

Reply:
addressed in the latest commit.